### PR TITLE
New Logo: Nextdoor

### PR DESCRIPTION
**Pull request by pablinos**
This adds the logo for Nextdoor. I've copied the procedure carried out to add Mastodon in #115. Building the assets, and bumping the version and adding a changelog entry.

### CODE REVIEW

File: svg-min-react/nextdoor.svg
Comment (pablinos) [inline svg-min-react/nextdoor.svg:1]: I'm not sure if this is what we want as the logos have turned out as just black circles! I notice that a similar thing happened with Mastodon though.

File: sources/svg/nextdoor.svg
Comment (jeherve) [inline sources/svg/nextdoor.svg:1]: I think we've sticked to black for previous logos. I think we should do the same here.

Comment (pablinos) [reply sources/svg/nextdoor.svg:1]: Ah yes, I'll fix that. Maybe the colours are part of the problem.

**Comment by pablinos** [comment]
I think I've managed to fix it. The SVG wasn't outlined properly. Would you mind taking another look when you get a chance please @jeherve?

**Review by jeherve — CHANGES_REQUESTED** [review]
I think you may have to update the lock file as well? Otherwise, this seems good to go!

**Review by renjunl — COMMENTED** [review]
Thank you for the effort in integrating ShareKit. I am the software engineer from Nextdoor Partnership team. Looking forward to collaborating with you in the future.

**Comment by pablinos** [comment]
> I think you may have to update the lock file as well? Otherwise, this seems good to go!

Oof, sorry, I've fixed that now, thanks @jeherve!

**Comment by pablinos** [comment]
> Thank you for the effort in integrating ShareKit. I am the software engineer from Nextdoor Partnership team. Looking forward to collaborating with you in the future.

Great, nice to meet you @renjunl! This change allows us to use the icon in our other code bases. I'll let you know as we raise more PRs

**Review by jeherve — APPROVED** [review]

**Comment by renjunl** [comment]
> > Thank you for the effort in integrating ShareKit. I am the software engineer from Nextdoor Partnership team. Looking forward to collaborating with you in the future.
> 
> Great, nice to meet you @renjunl! This change allows us to use the icon in our other code bases. I'll let you know as we raise more PRs

Thank you, can you tag @kevinchai0 kchai@nextdoor.com  in the future PRs related to Nextdoor too? Thanks.

**Comment by jeherve** [comment]
> I think I've managed to fix it. The SVG wasn't outlined properly.

when you get the chance, could you walk me through the steps you followed to do that, or take a look at the Mastodon and Titktok alt logos in this repo and see if the same fix can be applied to those logos too? That may allow us to move forward with #117.

Thank you!

**Review by jeherve — APPROVED** [review]
I've rebased things, it should be good to go now! Merging.

**Comment by pablinos** [comment]
Thanks @jeherve!

> when you get the chance, could you walk me through the steps you followed to do that, or take a look at the Mastodon and Titktok alt logos in this repo and see if the same fix can be applied to those logos too? 

It looks like they're working ok now, so I assume you sorted it, but the problem with this one was that the 'n' wasn't cut out from the background, so I outlined it. We can do the same to any other problematic ones.

**Comment by mikeritter** [comment]
Thanks guys!

**Pull request merged**